lib/data/sql/V1.1__import_lu.sql

```diff
@@ -27,8 +27,3 @@ CREATE TABLE lu_bestockung (STO_Nr TEXT, Kategorie TEXT, Fi TEXT, Ta TEXT, WFö
 COPY lu_bestockung
 FROM '/data/profiles/lu/bestockung.csv'
 DELIMITER ';' CSV HEADER;
-
-CREATE TABLE lu_uebergaenge (STO_Nr_nais TEXT, STO_Nr_profile TEXT);
```

Comment:
`lu_uebergaenge` brauchen wir nicht mehr?

Reply:
Das wurde inzwischen alles in die tree-lib verlegt